src/utils/dom.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * HtmlElement's attribute that can be set
+ */
+type HtmlElementAttributes = Pick<HTMLElement, 'contentEditable' | 'innerHTML'>;
```

Comment:
why only these attributes? I guess this helper should allow adding any attributes